modules/nf-core/plink/genome/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process PLINK_GENOME"
+    script "../main.nf"
+    process "PLINK_GENOME"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "plink"
+    tag "plink/genome"
+
+    
+    test("plink - genome") {
+        when {
+            params {
+                outdir = "test"
+            }
```

Comment:
```suggestion
```
No need for this 😄, `params.outdir` is mostly used on a pipeline level.

Reply:
Thank you, I've fixed this